src/test/java/com/placelab/tests/SmokeTest.java

```diff
@@ -0,0 +1,29 @@
+package com.placelab.tests;
+
+import com.placelab.utils.WebDriverSetup;
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+public class SmokeTest {
+    private WebDriver driver;
+
+    @BeforeTest
+    public void setup() {
+        driver = WebDriverSetup.getWebDriver(System.getProperty("browser"));
+        driver.get("https://demo.placelab.com/");
+
+        System.out.println("Opened browser");
+    }
+
+    @Test
+    public void openPage() {
+        System.out.println("Opened browser " + driver);
+    }
+
+    @AfterTest
+    public void printTitle() {
+        System.out.println(driver.getTitle());
+    }
```

Comment:
This printout could have been a part of the method with the @Test annotation. The AfterTest annotation should be used for closing the web browser, once we're done with executing whatever we have in the methods under the @Test annotations. If we do not close the browser in our tests, we are obliged to close it manually.